client/login/magic-login/index.jsx

```diff
@@ -137,11 +137,16 @@ class MagicLogin extends Component {
 	renderLinks() {
 		const { isJetpackLogin, locale, showCheckYourEmail, translate, isWoo, query } = this.props;
 
+		const isA4A = query?.redirect_to?.includes( 'agencies.automattic.com' ) ?? false;
```

Comment:
I'm wondering if we should do this only for client login:

agencies.automattic.com/client

Reply:
Thanks. The testing link now {live-link}/log-in/link?redirect_to=%22agencies.automattic.com%2Fclient%22